packages/app/routes.tsx

```diff
@@ -65,6 +66,7 @@ const routes = (
       <Route path="agentrelationshiprole" component={AgentRelationshipRoles} />
       <Route path="agentrelationship" component={AgentRelationship}/>
       <Route path="process" component={Process}/>
+      <Route path="plan" component={Plan}/>
       <Route path="person" component={PersonPage} />
     </Route>
   </Route>
```

Comment:
The "export default routes" needs a semicolon

Reply:
fixed